interface.html

```diff
@@ -364,12 +364,35 @@ <h5>Applies to</h5>
           <hr />
           <div class="row">
             <div class="col-md-4">
-              <h5>Columns to exclude</h5>
+              <h5>Include or exclude columns</h5>
               <p data-exclude-description></p>
             </div>
             <div class="col-md-8 side">
               <div class="tokenfield-ui">
-                <input type="text" class="form-control" name="exclude" value="" placeholder="Type the column name and press return">
+                <div class="row">
+                  <div class="col-md-3">
+                    <div class="radio radio-icon">
+                      <input type="radio" id="excludes" name="includes-excludes" value="false">
```

Comment:
```suggestion
                      <input type="radio" id="exclude" name="column-list-mode" value="exclude">
```

1. `include` or `exclude` doesn't need the `s`
2. `includes-excludes` looks like you're using all the possible values to name a field instead of naming the variable. If we one day add a 3rd value called `banana` then the `name` becomes `include-exclude-banana`?

Reply:
@tonytlwu  changes made by @NikitaJadhavXoriant  as per suggestion